endreprofil.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * TODO: Make the new crime script working again, but better!
+ */

```

Comment:
Why is this comment about the crime script here? "Todo"'s are plans on what to do next.